i_scene_cp77_gltf/__init__.py

```diff
@@ -44,6 +44,7 @@ def execute(self, context):
 class CP77Import(bpy.types.Operator,ImportHelper):
     bl_idname = "io_scene_gltf.cp77"
     bl_label = "Import glTF"
+    bl_description = "Load glTF 2.0 files with Cyberpunk 2077 materials" #Kwek: tooltips towards a more polished UI.
```

Comment:
Place a period at the end the description

Reply:
As mentioned in my comment above. If I add a period the tooltip will say this:

**Load glTF 2.0 files with Cyberpunk 2077 materials..**

..double period. 

..and it's my pleasure! I'm a fan of this plugin and happy to contribute :)